packages/twenty-front/src/constants/EmptyMutation.ts

```diff
@@ -0,0 +1,7 @@
+import gql from 'graphql-tag';
+
+export const EMPTY_MUTATION = gql`
```

Comment:
this should not leave in root folder. I would put it a graphql or in object-metadata as I think it's mainly used there for now